src/components/Forms/AllySignUpForm.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import {
+  Flex,
+  FormControl,
+  FormLabel,
+  Input,
+  Select,
+  useTheme,
+  Text,
+} from '@chakra-ui/core';
+
+import PrimaryButton from '../Buttons/PrimaryButton';
+import { submitAlly } from '../../services/AirtableServices';
+
+const skillTypes = [
```

Comment:
This is proving really difficult. I've looked through all the docs and as far as I can see they don't provide access to the select options. There are quite a few complaints about it in the community forums lol. I made a custom hook that grabs the first page of docs and filters for just the specialty field and then I reduce it to only unique items. But this really isn't performant. Not sure what you wanna do with it so I'll leave it in just in case.